doc/release/release_dev.rst

```diff
@@ -16,6 +16,10 @@ New Features
 ------------
 
 
+A new doc tutorial presenting a cell biology example has been added to the
+gallery (#4648). The scientific content benefited from a much appreciated
+review by Pierre Poulain and Fred Bernard, both assistant professors at
+Université de Paris and Institut Jacques Monod.
```

Comment:
```suggestion
- A new doc tutorial presenting a cell biology example has been added to the
  gallery (#4648). The scientific content benefited from a much appreciated
  review by Pierre Poulain and Fred Bernard, both assistant professors at
  Université de Paris and Institut Jacques Monod.
```

Reply:
@jni I wrote it as a paragraph to duplicate this format: https://github.com/scikit-image/scikit-image/blob/master/doc/release/release_0.17.rst#new-features (that @emmanuelle linked to). There is a list of 'actual' new features (as list items / bullet points), but the 'list' of doc / gallery additions is written in a paragraph... Are we following a definite convention then? Thanks!

Reply:
@mkcor the usual format is to have bullet points, for the last version I had just "bunched together" a few items in a paragraph. But I think this PR deserves its own bullet point as suggested by @jni.

Reply:
All right!